examples/distributed_simulation/README.md

```diff
@@ -0,0 +1,99 @@
+# Distributed Large Scale Simulation
+
+> **WARNING:**
+> **This example will consume a huge amount of tokens.**
+> **Please do not use model API that requires payment.**
```

Comment:
A little bit misleading. "Using model API with this example can introduce high cost." should be enough as a warning.